README.md

```diff
@@ -5,10 +5,14 @@ The WSO2 SAML SDK for Java enables software developers to integrate SAML based S
   single sign-on and federated access control solutions with minimum hassle.
 
 ## Trying out the sample
+
+### Prerequisites
+1. WSO2 Identity Server and it's [prerequisites](https://is.docs.wso2.com/en/next/setup/installing-the-product/).
+
 A sample app for demonstrating SAML based SSO authentication, SLO and attribute retrieval is hosted at:
 https://github.com/wso2-extensions/identity-agent-sso/tree/master/resources/SampleApp
 
-You can download the pre-built SampleApp.war from here[link].
+You can download the pre-built SampleApp.war from https://github.com/wso2-extensions/identity-agent-sso/releases/latest
```

Comment:
Can't we add a link which would download the webapp from the assets instead of redirecting the user to the releases page? Something like https://github.com/wso2-extensions/identity-agent-sso/releases/download/v5.5.5/SampleApp-5.5.5.war

Reply:
The version will change from time to time.
So, we will have to update the doc each time.

Reply:
@darshanasbg WDYT?

Reply:
Lets not use the version when buding the war file since this will impact the other instructions like URL to access the application etc.